src/components/TimePicker/TimePicker.tsx

```diff
@@ -560,18 +557,14 @@ function TimePicker({forwardedRef, defaultValue, onSubmit, onInputChange}) {
     );
 }
 
-TimePicker.propTypes = propTypes;
-TimePicker.defaultProps = defaultProps;
 TimePicker.displayName = 'TimePicker';
 
-const TimePickerWithRef = React.forwardRef((props, ref) => (
+const TimePickerWithRef = React.forwardRef((props: Omit<TimePickerProps, 'forwardedRef'>, ref: ForwardedRef<MinHrRefs>) => (
     <TimePicker
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
         forwardedRef={ref}
```

Comment:
Please check how other components do this, but you can remove `forwardedRef` and instead use `ref` prop, this way you won't need TimePickerWithRef component